collections/_release_4_4/entry-highlights-initial-android-editor-support-for-XR-devices.md

```diff
@@ -0,0 +1,21 @@
+---
+type: entry
+section: highlights
+rank: 2
+importance: 1
+anchor: "initial-android-editor-support-for-XR-devices"
+title: "Initial Android editor support for XR devices"
+blockquote: "Make games in VR!"
+text: |
+  Because the Godot Editor itself is a project made with the engine like any other, it can be made available in more unconventional places like the web or on mobile.
+
+  With this release, the first steps have been taken to use OpenXR to transfer the existing Android editor into the context of mixed reality headsets.
```

Comment:
Not only "mixed reality" but also "virtual reality". Usually, to cover both we write "XR":

```suggestion
  With this release, the first steps have been taken to use OpenXR to transfer the existing Android editor into the context of XR headsets.
```